src/domains/checkout-app/api/checkout/checkout_ecommerce/v1/_openapi.json.tpl

```diff
@@ -1168,6 +1174,28 @@
           "threeDsData": "threeDsData"
         }
       },
+      "CardsAuthRequestDetails": {
+        "type": "object",
+        "description": "Additional payment authorization details for cards NPG authorization",
+        "properties": {
+          "detailType": {
+            "description": "property discriminator, used to discriminate the authorization request detail. Fixed value 'cards'",
+            "type": "string"
+          },
+          "sessionId": {
```

Comment:
Add **bin** field into request for execute double check.

Reply:
Is this double check needed? Bin is retrieved from NPG response (cached or not) so maybe a check against bin would be useless, what do you think?

Reply:
Ok. I think your observation is correct 🙂